terra2/assetlist.json

```diff
@@ -486,6 +486,28 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/nicodao.png"
         }
       ]
-    }
-  ]
-}
+    },
+      {
+        "description": "Sailing the Seas DAO is a community DAO built on Enterprise on Terra",
+        "type_asset": "cw20",
+        "address": "terra1rc6ssp5rym7a0hg29xpj4cc9e67tl56kg5jyzgl9qrhfxxc2ugvsnrkala",
+        "denom_units": [
+          {
+            "denom": "cw20:terra1rc6ssp5rym7a0hg29xpj4cc9e67tl56kg5jyzgl9qrhfxxc2ugvsnrkala",
+            "exponent": 0
+          },
+          {
+            "denom": "seas",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:terra1rc6ssp5rym7a0hg29xpj4cc9e67tl56kg5jyzgl9qrhfxxc2ugvsnrkala",
+        "name": "Sailing The SeaS DAO",
+        "display": "seas",
+        "symbol": "SEAS",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/SEAS.png"
```

Comment:
this image doesn't exist?

Reply:
Please upload the image into that terra2/images/ folder in your branch

Reply:
![SEAS](https://github.com/cosmos/chain-registry/assets/143271799/616a6751-b993-4336-af6d-86374a497222)

I cannot could you do it  please ? link provided above

Reply:
I get an error uploads are disabled .

Reply:
File uploads require push access to this repository

Reply:
your branch is on your fork, so you should be able to

Reply:
Nope access denied 
![image](https://github.com/cosmos/chain-registry/assets/143271799/e8aa96fa-4762-493c-8a94-f73fe4c676c6)

Reply:
Which repo and branch is that where uploads are disabled?

Reply:
Cosmo/chain-registry/terra2/images

Thank You

Reply:
That's the wrong location, as you don't have push permissions to the cosmos' repo. You should be trying to upload it to your forked repo, which you have full permissions over.
https://github.com/sk3ngy/chain-registry/tree/patch-1/terra2/images

Reply:
All done :) Image upload successful thank you for your help.

Reply:
It's not there. Did you Commit after upload?

Reply:
Can you add it to your own patch-1 branch so it's added in this same PR?

Reply:
Done Thank You